packages/certificate-model/project.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "certificate-model",
+  "$schema": "../../node_modules/nx/schemas/project-schema.json",
+  "sourceRoot": "packages/certificate-model/src",
+  "projectType": "library",
+  "targets": {
+    "build": {
+      "executor": "@nx/js:tsc",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "outputPath": "dist/packages/certificate-model",
+        "tsConfig": "packages/certificate-model/tsconfig.lib.json",
+        "packageJson": "packages/certificate-model/package.json",
+        "main": "packages/certificate-model/src/index.ts",
+        "assets": ["packages/certificate-model/*.md"]
+      }
+    },
+    "publish": {
+      "command": "node tools/scripts/publish.mjs certificate-model {args.ver} {args.tag}",
```

Comment:
this default publish command is not really ergonomic, could replace it with this [plugin](https://github.com/bikecoders/ngx-deploy-npm) instead ?
you can then remove the `publish.mjs` script.

Reply:
Done. `lerna version` has obviously been removed, should something like https://github.com/jscutlery/semver be added to take care of versioning? I see you used it in your `nx-heroku` project.

Reply:
yes, that’s the same `Nx Plugin` i mentioned in the comment above.